src/c++/perf_analyzer/client_backend/client_backend.cc

```diff
@@ -117,6 +124,7 @@ ClientBackend::Create(
     const BackendKind kind, const std::string& url, const ProtocolType protocol,
     const GrpcCompressionAlgorithm compression_algorithm,
     std::shared_ptr<Headers> http_headers, const bool verbose,
+    const std::shared_ptr<TritonLoader>& loader,
```

Comment:
The TritonLoader should live in triton_c_api so it doesn't need to be here.

Reply:
@deadeyegoodwin 
the backend gets initialized every time a `Infer` is requested https://github.com/triton-inference-server/client/blob/762ef8290a311402517cc31992406dcb975c50c8/src/c%2B%2B/perf_analyzer/request_rate_manager.cc#L188
Since we are initializing the server from the capi, I thought having a separate instance always running is better

Reply:
I agree that you want to share on TRITONSERVER_Server object for all inference requests. But you should do this by using a singleton pattern (or similar) in the triton_c_api backend so that no matter how many triton_c_api backends are created they all use the same TRITONSERVER_Server object. Probably the easiest way to do this is encapsulate the singleton pattern in your TritonLoader class. I can provide more detail on this is my explanation is not clear.